DfE.FindInformationAcademiesTrusts/Pages/Trusts/Contacts/_EditContactsForm.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@model EditContactModel
+
+<fieldset class="govuk-fieldset">
+  <div class="govuk-form-group @Model.GetErrorClass("Name")">
```

Comment:
In the design system, the div is outside the fieldset. I don't know if this makes any kind of difference though